demo/.storybook/main.ts

```diff
@@ -2,7 +2,7 @@ import type { StorybookConfig } from "@storybook/web-components-vite";
 
 const config: StorybookConfig = {
   stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
-  addons: ["@storybook/addon-essentials"],
+  addons: ["@storybook/addon-docs"],
```

Comment:
Is `@storybook/addon-essentials` going away?

Reply:
All part of core now!